apis/apps/v1alpha1/imagepulljob_types.go

```diff
@@ -37,6 +37,11 @@ type ImagePullJobSpec struct {
 	// +optional
 	Selector *NodeSelector `json:"selector,omitempty"`
 
+	// PodSelector is a query over pods that should pull image on nodes of these pods.
+	// Mutually exclusive with Selector.
+	// +optional
+	PodSelector *metav1.LabelSelector `json:"podSelector,omitempty"`
```

Comment:
can we limit the pod selection by namespace in addition? select pods directly by selector can be very cpu consuming